src/components/Button/SubmitButton/SubmitButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { ReactNode } from "react";
+import styled from "styled-components";
+import COLORS from "styles/palette";
+import DefaultButton from "../Button.style";
+
+interface Props {
+  isActive: boolean;
+  children: ReactNode;
+  round?: boolean;
+  handleSubmit?: () => void;
+}
+
+export default function SubmitButton({
+  isActive,
+  children,
+  round = false,
+  handleSubmit = () => {},
+}: Props) {
+  return (
+    <StyledSubmitButton
+      type="button"
+      $isActive={isActive}
+      disabled={!isActive}
+      $round={round}
+      onClick={(e) => {
+        e.preventDefault();
+        handleSubmit();
+      }}
+    >
+      {children}
+    </StyledSubmitButton>
+  );
+}
+
+const StyledSubmitButton = styled.button<{
+  $isActive: boolean;
+  $round?: boolean;
+}>`
+  ${DefaultButton};
+  background-color: ${({ $isActive }) =>
```

Comment:
styled components를 사용할때 이렇게 props를 여러번에 걸쳐 가져오는것이 다 자바스크립트 함수를 여러번 실행하는거라고 보시면 돼요. 따라서 되도록이면 props를 한꺼번에 가져와서 css를 사용해 스타일을 적용해주는편이 좋겠죠? 

```suggestion
 ${({ $isActive, $round }) => css`
     background-color: ...
 `
```